src/components/GoogleAddressLookup/component.js

```diff
@@ -0,0 +1,297 @@
+                    }
+                    return resolve(predictions);
+                },
+            );
+        });
+    }
+
+    processSearchResults(results, searchValue) {
+        this.setState({
+            places: results,
+            suggestions: getSuggestions(results, searchValue),
+        });
+    }
+
+    handleError(searchValue) {
+        this.setState({
+            places: [],
+            suggestions: [getSearchForOption(searchValue)],
+        });
+    }
+
+    clear() {
+        this.setState({
+            places: [],
+            suggestions: [],
+        });
+    }
+
+    render() {
+        const {
+            style,
+            label,
+            error,
+            placeholder,
+            disabled,
+            tabIndex,
+            required,
+            id,
+            name,
+            value,
+            hideLabel,
+            onClick,
+            onFocus,
+            onBlur,
+        } = this.props;
+        const { isSearching, suggestions } = this.state;
+        const options = suggestions.length > 0 ? suggestions : null;
+        return (
+            <div id={id} style={style} className={this.getContainerClassName()}>
+                <Lookup
+                    id={this.lookupId}
+                    name={name}
+                    label={label}
+                    debounce
+                    isLoading={isSearching}
+                    placeholder={placeholder}
+                    options={options}
+                    hideLabel={hideLabel}
+                    readOnly={false}
+                    required={required}
+                    tabIndex={tabIndex}
+                    disabled={disabled}
+                    value={getFormattedValue(value, false, <SelectedLocationIcon />)}
+                    onChange={this.handleChange}
+                    onSearch={this.handleSearch}
+                    onClick={onClick}
+                    onFocus={onFocus}
+                    onBlur={onBlur}
+                    error={error}
+                    icon={<LocationIcon />}
+                />
+                {!error && (
+                    <div className="rainbow-google-address-lookup_poweredby-container">
+                        <PoweredByGoogleLogo className="rainbow-google-address-lookup_poweredby-logo" />
+                    </div>
+                )}
+            </div>
+        );
+    }
+}
+
+PlacesLookupComponent.propTypes = {
+    isScriptLoaded: PropTypes.bool.isRequired,
+    isScriptLoadSucceed: PropTypes.bool.isRequired,
+    searchOptions: PropTypes.shape({
+        bounds: PropTypes.shape({
+            sw: PropTypes.shape({
+                latitude: PropTypes.number.isRequired,
+                longitude: PropTypes.number.isRequired,
+            }),
+            ne: PropTypes.shape({
+                latitude: PropTypes.number.isRequired,
+                longitude: PropTypes.number.isRequired,
+            }),
+        }),
+        country: PropTypes.string,
```

Comment:
this can be an array of up to five country code strings too
we need to explain this in proptype description in index.js and mention example of use

Reply:
we need to check this... in the api reference i have the country parameter is just an string... maybe the new API changed